Cargo.toml

```diff
@@ -18,6 +18,7 @@ members = [
 ]
 
 [patch.crates-io]
+arbitrary = { git = "https://github.com/michaelsproul/arbitrary", rev="f002b99989b561ddce62e4cf2887b0f8860ae991" }
```

Comment:
why arbitrary here should be overrided?

Reply:
Because lighthouse requires that.

References:
- [The upstream did that.](https://github.com/sigp/lighthouse/blob/v4.1.0/Cargo.toml#L105)
- [So, our fork did that, too.](https://github.com/synapseweb3/lighthouse/blob/2c246d68011f14a9ee959b8d75b1dcb5d30b2993/Cargo.toml#L98)